internal/storage/v2/elasticsearch/tracestore/from_dbmodel.go

```diff
@@ -32,110 +31,21 @@ const (
 var errType = errors.New("invalid type")
 
 // ProtoToTraces converts multiple Jaeger proto batches to internal traces
-func ProtoToTraces(batches []*model.Batch) (ptrace.Traces, error) {
+func ProtoToTraces(spans []*dbmodel.Span) (ptrace.Traces, error) {
```

Comment:
* please use new function names, e.g. FromDbModel
* The core reader/writer API was changed to avoid pointers, so `[]dbmodel.Span` (can be separate PR)